schemas/telemetry/configuration-schema.json

```diff
@@ -112,6 +112,10 @@
                   "type": "boolean",
                   "description": "Whether it is allowed to use LocalStorage when cookies are not available"
                 },
+                "store_contexts_across_pages": {
+                  "type": "boolean",
+                  "description": "Whether contexts are stored in local storage"
+                },
```

Comment:
The description and name don't really say the same thing. Does the `store_contexts_across_pages` feature necessarily means using local storage?

Reply:
Indeed, my intent was to add a bit more details than what is already conveyed by the name of the configuration options.
"Across pages" is more user facing and "in local storage" is more from a technical point of view.
We only have local storage for now, no worries to match name and description if it is disturbing.